gpiozero/boards.py

```diff
@@ -1532,6 +1532,38 @@ def __init__(self, pwm=False, pin_factory=None):
         )
 
 
+class TrafficpHAT(LEDBoard):
+    """
+    Extends :class:`LEDBoard` for the `Pi Supply Traffic pHAT`_: Mini traffic
+    light LEDs.
+
+    The Traffic pHAT pins are fixed and therefore there's no need to specify
+    them when constructing this class. The following example then turns on all the LEDs::
+
+        from gpiozero import TrafficpHAT
+        hat = TrafficHat()
+        hat.lights.on()
+
+    :param bool pwm:
+        If :data:`True`, construct :class:`PWMLED` instances to represent each
+        LED. If :data:`False` (the default), construct regular :class:`LED`
+        instances.
+
+    :type pin_factory: Factory or None
+    :param pin_factory:
+        See :doc:`api_pins` for more information (this is an advanced feature
+        which most users can ignore).
+
+    .. _PiSupply Traffic pHAT:
```

Comment:
missing link

Reply:
I've added a placeholder link which will redirect when it's launched. :+1: